### ISSUE TITLE
Next JS zorunlu mu?

### ISSUE THREAD

**Issue by Fairlanders** [SelmanKahya/mukemmel-blog#9]
Sadece JS,HTML,CSS(SCSS)'de yazsak olmuyor mu yoksa illa ki bir application mı olacak bu blog? Videoda tam bir açıklama bulamadım nextjsle yapıyorsun sen ama biz de mi yapacağız?

**Comment by selmank-front** [comment]
evet, yarismaya katilmak icin nextjs kullanarak yapmalisin. boylelikle katilimcilari birbiriyle objektif bir sekilde kiyaslayabilirim.
